contracts/hardhat.config.cjs

```diff
@@ -153,4 +153,40 @@ task('set-burn-fee-ratio', 'Change the burn fee ratio')
     console.log(`Burn Fee Ratio changed from:${curBurnFeeRatio} to ${newBurnFeeRatio}`)
   })
 
+task('send-klay', 'Send $KLAY from faucet')
+  .addParam('address', 'The receiving address')
+  .addParam('amount', 'The amount of $KLAY to be send')
+  .setAction(async (taskArgs, hre) => {
+    let wallet
+    const PROVIDER = process.env.PROVIDER || ''
```

Comment:
I guess, the `|| ''` is not necessary as other hardhat tasks do not have it as well. Additionally, it does not help with not providing the necessary environment variable.